dev/com.ibm.ws.threading/src/com/ibm/ws/threading/internal/ThreadPoolController.java

```diff
@@ -665,6 +703,9 @@ enum LastAction {
         String tpcPoolIncrementBoundMedium = getSystemProperty("tpcPoolIncrementBoundMedium");
         poolIncrementBoundMedium = (tpcPoolIncrementBoundMedium == null) ? poolChangeBasis * 64 : Integer.parseInt(tpcPoolIncrementBoundMedium);
 
+        poolIncrementBoundHigh = poolIncrementBoundMedium * 2;
+        poolIncrementBoundExtraHigh = poolIncrementBoundHigh * 2;
```

Comment:
Yes that was intentional - we have system properties for the smaller poolIncrement components which feed into these, so we could damp down the increment rate with those if needed. It would not hurt to have props for these settings also, but seemed like it would not add much, and there are many triage points already.